src/main/java/de/tum/cit/aet/artemis/communication/domain/AnswerPost.java

```diff
@@ -35,10 +39,17 @@ public class AnswerPost extends Posting {
     @OneToMany(mappedBy = "answerPost", cascade = CascadeType.REMOVE, orphanRemoval = true, fetch = FetchType.EAGER)
     private Set<Reaction> reactions = new HashSet<>();
 
+    @OneToMany(mappedBy = "postId", cascade = CascadeType.REMOVE, orphanRemoval = true, fetch = FetchType.EAGER)
+    @SQLRestriction("post_type = 1")
```

Comment:
I unfortunately do not know how I would use the enum value there, since the server starts screaming as soon as I try to concat it into the SQLRestriction. However, I tried to add a comment so that developers know where the number comes from. 👍